nsqadmin/static/js/views/app.js

```diff
@@ -136,6 +136,10 @@ var AppView = BaseView.extend({
     },
 
     onLinkClick: function(e) {
+        if (e.ctrlKey || e.metaKey) {
+          // allow ctrl+click to open in a new tab
+          return true
```

Comment:
fixed to `return;` I don't think we actually need `stopPropagation` so going to choose to omit that.